v2/concordium/protocol-level-tokens.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+package concordium.v2.plt;
+
+// This specifies the package we want to use for our generated Go code.
+// Has no effect on code generated on other languages.
+option go_package = "./pb";
+
+// This specifies the package we want to use for our generated Java classes.
+// Has no effect on code generated on other languages.
+option java_package = "com.concordium.grpc.v2.plt";
+
+// This specifies that separate .java files will be generated for each of the Java classes/enums/etc. generated for the top-level messages, services, and enumerations,
+// and the wrapper Java class generated for this .proto file won't contain any nested classes/enums/etc.
+// If not generating Java code, this option has no effect.
+option java_multiple_files = true;
+
+// This specifies the package we want to use for our generated C# classes.
+// Has no effect on code generated on other languages.
+option csharp_namespace = "Concordium.Grpc.V2.Plt";
+
+import "kernel.proto";
+
+// The unique symbol and identifier of a protocol level token.
+message TokenId {
+  // Unique identifier for the token, guaranteed to be distinct across the
+  // entire concordium blockchain.
+  string symbol = 1;
+}
```

Comment:
Do we know the invariants for this type already, or are they still to be determined? The reason I ask that here is I'm wondering why this is not just a string instead?

Reply:
I think it makes sense that this is a newtype instead to distinguish it from other strings. I expect the symbol strings will follow a certain schema (undetermined at the moment). And they also need to have the uniqueness property across the chain.

Reply:
Yeah that makes sense 🙂